Source/DataStructureAndEncodingDefinition/gdcmPreamble.h

```diff
@@ -63,7 +63,6 @@ class GDCM_EXPORT Preamble
     }
   Preamble& operator=(Preamble const &)
     {
-    Internal = nullptr;
     Create();
     return *this;
```

Comment:
shouldn't we also copy characters from Internal?

Reply:
To hollow the current logic, we should not. Both copy operator and copy assign operator do not actually copy any data but fill allocated memory in some curtain fixed way.  Moreover, these operators do not have an explicit input object. 

Thus, let's wait for author's response.  

Anyway, it not a big deal to add true copying, if it will be needed.